test/integration/nft.test.ts

```diff
@@ -30,7 +30,8 @@ describe('E2E integration tests', () => {
   beforeAll(async () => {
     await loadAlchemyEnv();
     alchemy = await new Alchemy({
-      apiKey: process.env.ALCHEMY_API_KEY
+      // apiKey: process.env.ALCHEMY_API_KEY
+      url: 'https://eth-mainnet.g.alchemypreview.com/nft/v3/demo'
```

Comment:
Probably fine for integration tests, but the demo key is sometimes overloaded and will 429. You can make a key specially for the sdk that won't suffer the same fate.

Reply:
oops meant to undo this change